specification/search/resource-manager/Microsoft.Search/preview/2024-03-01-preview/examples/SearchListOperations.json

```diff
@@ -0,0 +1,616 @@
+                    "displayName": "Search Latency",
+                    "displayDescription": "Average search latency for the search service",
+                    "unit": "Seconds",
+                    "aggregationType": "Average",
+                    "availabilities": [
+                      {
+                        "timeGrain": "PT1M",
+                        "blobDuration": "PT1H"
+                      }
+                    ]
+                  },
+                  {
+                    "name": "SearchQueriesPerSecond",
+                    "displayName": "Search queries per second",
+                    "displayDescription": "Search queries per second for the search service",
+                    "unit": "CountPerSecond",
+                    "aggregationType": "Average",
+                    "availabilities": [
+                      {
+                        "timeGrain": "PT1M",
+                        "blobDuration": "PT1H"
+                      }
+                    ]
+                  },
+                  {
+                    "name": "ThrottledSearchQueriesPercentage",
+                    "displayName": "Throttled search queries percentage",
+                    "displayDescription": "Percentage of search queries that were throttled for the search service",
+                    "unit": "Percent",
+                    "aggregationType": "Average",
+                    "availabilities": [
+                      {
+                        "timeGrain": "PT1M",
+                        "blobDuration": "PT1H"
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "name": "Microsoft.Search/searchServices/logDefinitions/read",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "The log definition for the search service",
+              "operation": "Read search service log definitions",
+              "description": "Gets the available logs for the search service"
+            },
+            "origin": "system",
+            "properties": {
+              "serviceSpecification": {
+                "logSpecifications": [
+                  {
+                    "name": "OperationLogs",
+                    "displayName": "Operation Logs",
+                    "blobDuration": "PT1H"
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "name": "Microsoft.Search/searchServices/privateEndpointConnectionProxies/validate/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Private Endpoint Connection Proxy",
+              "operation": "Validate Private Endpoint Connection Proxy",
+              "description": "Validates a private endpoint connection create call from NRP side"
```

Comment:
I don't know what NRP means and I googled it with private endpoint connection and I still couldn't find an answer.